packages/twenty-zapier/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "twenty-zapier",
-  "version": "1.0.2",
+  "version": "2.0.0",
```

Comment:
2.0 ? :p should we follow the twenty versions for now: 0.32.0 ?

Reply:
It is a breaking change update, so I need to update X in X.Y.Z